src/components/cards/cards.css

```diff
@@ -0,0 +1,52 @@
+.cards-wrapper {
```

Comment:
Cards component should be using responsive design.

![image](https://user-images.githubusercontent.com/48954743/226596627-df4fc184-f29f-40b8-817e-32f8e7baecf0.png)